src/_griffe/loader.py

```diff
@@ -261,8 +261,7 @@ def resolve_aliases(
                 resolved |= next_resolved
                 unresolved |= next_unresolved
             logger.debug(
-                "Iteration %s finished, {len(resolved)} aliases resolved, still {len(unresolved)} to go",
-                iteration,
+                f"Iteration {iteration} finished, {len(resolved)} aliases resolved, still {len(unresolved)} to go",
```

Comment:
Lets use the %-format instead :slightly_smiling_face: Best used with logging, for perf reasons. Not more performant at all here, but for the sake of consistency.

```suggestion
                "Iteration %s finished, %s aliases resolved, still %s to go",
                iteration,
                len(resolved),
                len(unresolved),
```